index.html

```diff
@@ -50,20 +50,11 @@ <h2>Skills</h2>
   <section id="projects">
     <h2>Projects</h2>
     <ul></ul>
-    
-
-  </section>
-  <section id="connect">
 
-    <h3>Connect</h3>
-    <ul>
-      <li> https://github.com/Jessica12492/intro-to-programming-dorado</li>
-      <li> https://mail.google.com</li>
-      <li> https://www.facebook.com</li>
 
-    </ul>
   </section>
 
+
   <section>
     <h2>Leave a Message</h2>
```

Comment:
Currently the title kind of sticks to the top of the form; would be nice to give it some breathing room just like you have for all the other sections

![Screen Shot 2023-01-31 at 2 02 05 PM](https://user-images.githubusercontent.com/9094098/215857371-43ceec3e-6b93-4f52-a786-59eb825d131b.png)


One way would be to add a margin to the form:

```css
form[name="leave_message"] {
    margin-top: 20px;
}
```